nativescript-core/ui/tabs/tabs.ios.ts

```
@@ -1036,19 +920,29 @@ export class Tabs extends TabsBase {
     }
 
     public getTabBarTextTransform(): TextTransform {
-        return null;
+        switch (this._ios.tabBar.titleTextTransform) {
+            case MDCTabBarTextTransform.None:
+                return "none";
+            case MDCTabBarTextTransform.Automatic:
+                return "initial";
+            case MDCTabBarTextTransform.Uppercase:
+            default:
+                return "uppercase";
```

Wouldn't it be better to throw an exception in the default branch? I think it would be easier to find out if for some reason there is a new property in the enumeration.

---

The thing is that this is something that is set through CSS and it works of Android for all the cases, but iOS doesn't support all, so I would prefer not to crash the app with a CSS change. On the other hand, if you cannot achieve what you want with a single style you can always use platform-specific ones.